test/extended/image_ecosystem/s2i_perl.go

```diff
@@ -50,6 +50,10 @@ var _ = g.Describe("[image_ecosystem][perl][Slow] hot deploy for openshift perl
 				err := oc.Run("new-app").Args("-f", dancerTemplate).Execute()
 				o.Expect(err).NotTo(o.HaveOccurred())
 
+				g.By("updating the config env to only run one httpd process")
+				err := oc.Run("set").Args("env", "dc/dancer-mysql-example", "HTTPD_START_SERVERS=1", "HTTPD_MAX_SPARE_SERVERS=1", "HTTPD_MAX_REQUEST_WORKERS=1")
+				o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
if you're curious this won't trigger an additional deployment because the image doesn't exist yet.  (unless the build magically runs incredibly fast, like sub-second fast, which isn't going to happen)

Reply:
@bparees Would it be cleaner to do it like @jim-minter did here (https://github.com/openshift/origin/pull/17578) to avoid confusion/the possibility of further issues?

Reply:
yeah i dunno why i thought it couldn't be done w/ the new-app command.  updated.  also fixed the err variable issue.